src/app/shared/models/shelf-configuration.ts

```diff
@@ -0,0 +1,12 @@
+import { Tv2ActionContentType } from './tv2-action'
+
+export interface ShelfConfiguration {
+  id: string
+  actionPanels: ShelfActionPanelConfiguration[]
+}
+
+export interface ShelfActionPanelConfiguration {
+  name: string
+  rank: number
+  actionFilter: Tv2ActionContentType[]
```

Comment:
I wouldn't know what actionFilter contained without looking at the type. It could be renamed to `allowedContentTypes`.

Reply:
I get your point. 
The filter needs to be updated once the Settings page is done. For now, I am only filtering on ContentType, but I am not sure that is going to be enough. 
ContentType won't, for example, allows us differentiate between different kinds of Graphics.